backend/LexBoxApi/GraphQL/LexQueries.cs

```diff
@@ -186,6 +186,22 @@ public IQueryable<User> UsersInMyOrg(LexBoxDbContext context, LoggedInContext lo
         return context.Users.Where(u => u.Organizations.Any(orgMember => myOrgIds.Contains(orgMember.OrgId)));
     }
 
+    [UseOffsetPaging]
+    [UseProjection]
+    [UseFiltering]
+    [UseSorting]
+    public IQueryable<User> UsersICanSee(LexBoxDbContext context, LoggedInContext loggedInContext)
+    {
+        var myOrgIds = loggedInContext.User.Orgs.Select(o => o.OrgId).ToList();
+        var myProjectIds = loggedInContext.User.Projects.Select(p => p.ProjectId).ToList();
+        var myManagedProjectIds = loggedInContext.User.Projects.Where(p => p.Role == ProjectRole.Manager).Select(p => p.ProjectId).ToList();
+        return context.Users.Where(u =>
+            u.Organizations.Any(orgMember => myOrgIds.Contains(orgMember.OrgId)) ||
+            u.Projects.Any(projMember =>
+                myManagedProjectIds.Contains(projMember.ProjectId) ||
+                (projMember.Project != null && projMember.Project.IsConfidential == false && myProjectIds.Contains(projMember.ProjectId))));
```

Comment:
I think we want `IsConfidential != true` because we decided to show users for unknown projects.

Reply:
Done in commit 44db0e32.